### PR TITLE
New term entry for Java Math.tan() 

### LINKED ISSUE #917: [Term Entry] Java Math Methods: `.tan()`
### Reason/inspiration (optional)

Under `java/concepts/math-methods/terms/`, create a new term entry for the `Math.tan()` Java method.

- In the `terms/` folder, make a new folder called `tan`
- Inside the `tan` folder, create a new markdown file called `tan.md`.
- Inside the file, be sure to include a "Syntax" section with a \```pseudo code snippet.
- Also make sure to include an "Example" section with a \```java code snippet.

Please refer to the [entry templates](https://github.com/Codecademy/docs/blob/main/documentation/), [content standards](https://github.com/Codecademy/docs/blob/main/documentation/content-standards.md), and [markdown style guide](https://curriculum-documentation.codecademy.com/content-guidelines/markdown-style-guide/) when working on the PR for this issue.

### Entry Type (select all that apply)

New Term for Existing Concept Entry

### Code of Conduct

- [X] I agree to follow Codecademy Doc's Code of Conduct

### For Maintainers

- [X] Labels added
- [X] Issue is assigned

### PR DESCRIPTION
**Pull request by shlok-2003**
### Description

Created documentation for `Math.tan()` function in java. It has a commented Codebyte example for future use.  The branch is named 'arguments'. Closes #917

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/java/concepts/math-methods/terms/tan/tan.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/tan/tan.md:3]:
```suggestion
Description: 'Returns the tangent of an angle given in radians'
```

File: content/java/concepts/math-methods/terms/tan/tan.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/tan/tan.md:15]:
```suggestion
The **`Math.tan()`** method returns tangent of an angle given in radians. The function accepts all real numbers *except* the values where `cos(x)` is equal to 0  
```

File: content/java/concepts/math-methods/terms/tan/tan.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/tan/tan.md:22]:
```suggestion

Here `x` is the argument passed to the function. It accepts all real numbers. 

```

File: content/java/concepts/math-methods/terms/tan/tan.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/tan/tan.md:28]:
```suggestion
This example uses `.tan()` to return the tangent for `(x)`:
```

File: content/java/concepts/math-methods/terms/tan/tan.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/tan/tan.md:34]: The number 45 seems more like an example given in degrees. Maybe do an example that's some multiple or fraction of PI.

File: content/java/concepts/math-methods/terms/tan/tan.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/tan/tan.md:49]: You can remove this comment.

**Review by SSwiniarski — CHANGES_REQUESTED** [review]
@shlok-2003, I did a first review and left a few comments.

**Comment by shlok-2003** [comment]
@SSwiniarski Do I have to do this from my side, and push it again? Or is there any other way ?

File: content/java/concepts/math-methods/terms/tan/tan.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/tan/tan.md:3]:
```suggestion
Description: 'Returns the tangent of an angle given in radians.'
```

File: content/java/concepts/math-methods/terms/tan/tan.md
Comment (SSwiniarski) [inline content/java/concepts/math-methods/terms/tan/tan.md:22]:
```suggestion
Here `x` is the argument passed to the function. It accepts all real numbers. 

```

**Review by SSwiniarski — APPROVED** [review]
@shlok-2003, I had two more minor comments. Otherwise looks good for review two,

**Comment by shlok-2003** [comment]
@SSwiniarski Required changes have been made

**Comment by shlok-2003** [comment]
@SSwiniarski I have made some more updates.

File: content/java/concepts/math-methods/terms/tan/tan.md
Comment (HishamT) [inline content/java/concepts/math-methods/terms/tan/tan.md:15]: "...returns *the* tangent of an angle given in radians."
"...is equal to zero."

File: content/java/concepts/math-methods/terms/tan/tan.md
Comment (HishamT) [inline content/java/concepts/math-methods/terms/tan/tan.md:22]: Place a comma after "Here". The word "it" in the second sentence is unclear as to what it is referring to. Consider this sentence instead:

"`Math.tan()` accepts all real numbers as arguments."

**Review by HishamT — CHANGES_REQUESTED** [review]
Hey @shlok-2003, I reviewed your entry. Left a few comments for you before approval. :smile:

**Comment by shlok-2003** [comment]
Hey @HishamT, I have made the changes in the files.

**Review by HishamT — APPROVED** [review]
LGTM!

**Comment by shlok-2003** [comment]
> LGTM!

hey @HishamT , is the pull request approved ?

**Comment by yangc95** [comment]
@shlok-2003 it looks like this PR has two approvals! When you get the chance, make sure to run the necessary scripts (we can also assist with this process or walk you through it) and update this PR branch.

**Comment by shlok-2003** [comment]
Hey @yangc95, I tried resolving the test by downloading yarn and using it. But, it didn't work for me, as I am not familiar with yarn. So if you help me with the functions and syntax of yarn, it would be of great help.

**Comment by yangc95** [comment]
Thanks for speeding up the process @Dusch4593!

@shlok-2003 live link: https://www.codecademy.com/resources/docs/java/math-methods/tan
Did you wish to remain anonymous for this?

cc: @HishamT @SSwiniarski

**Pull request merged**
